src/cljs/org/broadinstitute/firecloud_ui/page/methods_configs_acl.cljs

```diff
@@ -21,11 +21,66 @@
 (def ^:private access-levels
   ["READER" "OWNER" "NO ACCESS"])
 
+(def ^:private reader-level (nth access-levels 0))
+(def ^:private  owner-level (nth access-levels 1))
+(def ^:private  no-access-level (nth access-levels 2))
+
+(defn- index-to-access-level [idx]
+  (str (nth access-levels idx)))
+
+(defn- access-level-to-index [access-level]
+  (case access-level
+    reader-level 0
+    owner-level 1
+    no-access-level 2))
+
 (def ^:private column-width "calc(50% - 4px)")
 
+(defn- correspondsToReader [access-level]
+  (if
+    (or
+      (= access-level reader-level)
+      (= access-level owner-level))
+    true
+    false))
+
+(defn- filter-public [acl-vec]
+  (let [hasNotPublicUser
+        (fn [m]
+          (not
+            (and
+              (map? m)
+              (contains? m :user)
```

Comment:
Do we really need to be testing whether the item is a map and conforms to the expected schema?  Seems like upstream code should guarantee this.